index.js

```diff
@@ -1731,12 +1742,12 @@
   //. Curries the given ternary function.
   //.
   //. ```javascript
-  //. > const replaceString = S.curry3 ((what, replacement, string) =>
-  //. .   string.replace (what, replacement)
-  //. . )
+  //. > const defineProperty = S.curry3 (Object.defineProperty)
```

Comment:
It is not easy to find a built-in function that takes three arguments!

Reply:
You could use a variadic function, such as `Math.max`, right? Then you could create curried built-ins for any number of arguments.

Reply:
That's true. Binary versions of variadic functions are compelling, but the use cases for ternary versions are less apparent. I am happy to have found `Object.defineProperty` for this example. :)